src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/config/UserContext.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.config
+
+import org.springframework.stereotype.Component
+
+@Component
+object UserContext {
+  private val authToken = ThreadLocal<String>()
```

Comment:
Should this have @RequestScope?

Reply:
You can't add @RequestScope to `objects` - it causes an error. I've copied this from whereabouts; however I believe this is inherently 'request scoped' because of ThreadLocal